next.config.js

```diff
@@ -13,6 +13,16 @@ const SentryWebpackPluginOptions = {
 }
 
 module.exports = withSentryConfig({
+  async redirects() {
+    return [
+      {
+        source: '/experimental/mat',
+        destination: '/chart/mat',
+        permanent: true,
```

Comment:
It's worth noting that this uses the 308 status code instead of 301 to implement the permanent redirect.

From what I can tell this is supported in most modern browsers, so it should be fine to use. Search engines also seem to prefer it over 301.